source/elements/oneDAL/source/common_interface/header_files_and_namespaces.rst

```diff
@@ -1,3 +1,61 @@
-===========================
-Header files and namespaces
-===========================
+============
+Header files
+============
+
+oneDAL public identifiers are represented in the following header files:
+
+.. list-table::
+   :widths:  10 50
+   :header-rows: 1
+
+   * - header file
+     - description
+   * - ``onedal/onedal.hpp```
+     - Main header file of oneDAL library.
+   * - ``onedal/algo/%ALGORITHM%/%ALGORITM%.hpp``
+     - An algorithm header file for a particular algorithm.
+       The string %ALGORITHM% should be substituted with the name of the algorithm, for example, ``kmeans`` or ``knn``.
```

Comment:
should %ALGORITHM%` be with rst markup?
```
``%ALGORITHM%``
```
same for utility, functionality, parent